app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,262 @@
+      ExTwitter::Client.new.mutual_friends(self)
+    end
+
+    def common_friends(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_friends(self, other)
+    end
+
+    def common_followers(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_followers(self, other)
+    end
+
+    def removing
+      return [] if TwitterUser.where(uid: uid, user_id: user_id).limit(2).pluck(:id).size < 2
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        ExTwitter::Client.new.removing(old_one, new_one)
+      end.flatten
+    end
+
+    def removed
+      return [] if TwitterUser.where(uid: uid, user_id: user_id).limit(2).pluck(:id).size < 2
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        ExTwitter::Client.new.removed(old_one, new_one)
+      end.flatten
+    end
+
+    def replying(options = {})
+      begin
+        client.replying(__uid_i, options.merge(tweets: statuses)).map { |u| u.uid = u.id; u }
+      rescue => e
+        logger.warn "#{self.class}##{__method__} #{e.class} #{e.message}"
+        []
+      end
+    end
+
+    def replied(options = {})
+      if ego_surfing? && mentions.any?
+        result = mentions.map { |m| m.user }.map { |u| u.uid = u.id; u }
+        (options.has_key?(:uniq) && !options[:uniq]) ? result : result.uniq { |u| u.id.to_i }
+      else
+        _client = ExTwitter::Client.new
+        uids = _client._extract_uids(search_results, options)
+        _client._extract_users(search_results, uids, options).map { |u| u.uid = u.id; u }
+      end
+    end
+
+    def favoriting(options = {})
+      client.favoriting(__uid_i, options.merge(favorites: favorites)).map { |u| u.uid = u.id; u }
+    end
+
+    def inactive_friends
+      ExTwitter::Client.new._extract_inactive_users(friends, authorized: ego_surfing?)
+    end
+
+    def inactive_followers
+      ExTwitter::Client.new._extract_inactive_users(followers, authorized: ego_surfing?)
+    end
+
+    def clusters_belong_to
+      text = statuses.map{|s| s.text }.join(' ')
+      ExTwitter::Client.new.clusters_belong_to(text)
+    end
+
+    def close_friends(options = {})
+      min = options.has_key?(:min) ? options.delete(:min) : 1
+      client.close_friends(__uid_i, options.merge(
+        min: min,
+        replying: replying(options.merge(uniq: false)),
+        replied: replied(uniq: false),
+        favoriting: favoriting(options.merge(uniq: false)))
+      ).map { |u| u.uid = u.id; u }
+    end
+
+    def inactive_friends_graph
+      inactive_friends_size = inactive_friends.size
+      friends_size = friends_count
+      [
+        {name: I18n.t('legend.inactive_friends'), y: (inactive_friends_size.to_f / friends_size * 100)},
+        {name: I18n.t('legend.not_inactive_friends'), y: ((friends_size - inactive_friends_size).to_f / friends_size * 100)}
```

Comment:
__[rubocop]__
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [124/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)